### PR TITLE
commands: Pull command to update repos

### PR DESCRIPTION
**Pull request by Attamusc**

### CODE REVIEW

File: lib/fig-newton/app.rb [hunk 1]
```diff
@@ -13,6 +13,13 @@ def clone(parent_directory = ".")
       `git clone #{full_repo} #{clone_directory} &>/dev/null`
     end
 
+    def pull(parent_directory = ".")
+      source_directory = full_dir(parent_directory)
+
+      puts "-- Pulling latest upstream in '#{source_directory}'"
+      `cd #{source_directory} && git pull`
```
Comment (chr0n1x) [inline lib/fig-newton/app.rb]: ugggh the amount that we shell out to git, you think we would've made a gem already

Comment (Attamusc) [inline lib/fig-newton/app.rb]: But but but. this is gonna eventually be in go :(

Comment (chr0n1x) [inline lib/fig-newton/app.rb]: doesn't need to happy, I just got sad when I thought about the N # of other of places where we do this

**Comment by chr0n1x** [comment]
:+1:

File: lib/fig-newton/app.rb (hunk 1 above)
Comment (Attamusc) [inline lib/fig-newton/app.rb]: Ya agreed :(

**Pull request merged**